libraries/c_sdk/standard/mqtt/src/iot_mqtt_subscription.c

```diff
@@ -44,6 +44,9 @@
 /* Platform layer includes. */
 #include "platform/iot_threads.h"
 
+/* Using initialized connToContext variable. */
+extern _connContext_t connToContext[ MAX_NO_OF_MQTT_CONNECTIONS ];
```

Comment:
Does not seem used in this file?

Reply:
The second PR for Phase 2 will use it, so I included it beforehand.